react/components/ProductPrice/index.js

```diff
@@ -128,10 +148,21 @@ Price.propTypes = {
   showInstallments: PropTypes.bool.isRequired,
```

Comment:
I think that this propTypes were big enough to be extracted to another js. `propTypes.js `